core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -32,6 +32,7 @@
 #include <impl/Kokkos_Atomic_View.hpp>
 #include <impl/Kokkos_Tools.hpp>
 #include <impl/Kokkos_StringManipulation.hpp>
+#include <impl/Kokkos_ZeroMemset_fwd.hpp>
```

Comment:
Similar question here, why is the fallback generic implementation in that file rather than in its own header?

Reply:
The compiler would complain if it couldn't see the default implementation at the point it is used. Yes, having a separate file for the fallback implementation makes sense to me but the question is if you can get the includes right to make that work.